appveyor.yml

```diff
@@ -0,0 +1,25 @@
+# Should we cache local modules ?
+# cache:
+#  - node_modules
+
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "7"
+    - nodejs_version: "8"
```

Comment:
@stof I agree, I just followed the specs as given by `package.json` which states >= 6.
I can change to only test on 6 if that is what we desire